src/cljs/nr/account.cljs

```diff
@@ -102,22 +82,14 @@
 
 (defn- remove-card-art
   [card s]
-  (swap! s update-in [:alt-arts] #(dissoc % (keyword (:code card))))
-  (when-let [replaces (:replaces card)]
-    (let [replaced-card (some #(when (= replaces (:code %)) %) (:cards @app-state))]
-      (when replaced-card
-        (remove-card-art replaced-card s)))))
+  (swap! s update-in [:alt-arts] #(dissoc % (keyword (:code card)))))
```

Comment:
```suggestion
  (swap! s update :alt-arts dissoc (keyword (:code card))))
```

A long long time ago, Clojure didn't have `update`, only `update-in`. Now we do! `update` takes a keyword, a function, and any number of arguments. It then `apply`s the function to the argument list, passing in the current value as the first argument, making it p much: `(apply dissoc (:alt-arts @s) (keyword (:code card)))`. This is purely a "style" thing, lol, but one of those that I've been trying to clean up in the codebase. If you don't care to update this, no worries.